trin-core/src/portalnet/types.rs

```diff
@@ -579,46 +511,22 @@ mod test {
     }
 
     #[test]
-    fn test_found_content_encodes_empty() {
-        let empty_enrs: Vec<SszEnr> = vec![];
-        let empty_payload: Vec<u8> = vec![];
-        let msg = FoundContent {
-            enrs: empty_enrs.clone(),
-            payload: empty_payload.clone(),
-        };
-        let actual = msg.as_ssz_bytes();
-        let decoded = FoundContent::from_ssz_bytes(&actual).unwrap();
-        assert_eq!(decoded, msg);
-        assert_eq!(decoded.enrs, empty_enrs);
-        assert_eq!(decoded.payload, empty_payload);
-        assert_eq!(actual.len(), msg.ssz_bytes_len());
-    }
-
-    #[test]
-    fn test_found_content_encodes_payload() {
-        let empty_enrs: Vec<SszEnr> = vec![];
-        let msg = FoundContent {
-            enrs: empty_enrs,
-            payload: vec![1; 32],
-        };
+    fn test_found_content_encodes_content() {
```

Comment:
nit: Given the name change from `FoundContent` to `Content`, it would be more clear to make that consistent in the names of the tests.